### PR TITLE
V1 Design | header favorites

### PR DESCRIPTION
**Pull request by Da-Colon**
## Description
Implements Favorites Menu

## Notes
-  useDisplayName hook will need to be updated when the DAO name register contract is implemented

## Screenshots (if applicable)
image

### CODE REVIEW

File: src/components/menus/FavoritesMenu/FavoritesList.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                  border: 'none',
+                  boxShadow: 'none',
+                  background: 'grayscale.500',
+                  borderRadius: '0.5rem',
+                  minHeight: '2.5rem',
+                },
+                '&::-webkit-scrollbar-thumb:hover': {
+                  backgroundColor: 'grayscale.300',
+                },
+              }}
```
Comment (Da-Colon) [inline src/components/menus/FavoritesMenu/FavoritesList.tsx:71]: default scroll looked really bad here. I will look into adding a class to the theme that can be used instead

**Comment by herbig** [comment]
@Da-Colon is there a way I can set a DAO as a favorite in this branch?

File: src/components/menus/FavoritesMenu/index.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          <Flex
+            gap="2"
+            alignItems="center"
+          >
+            <StarOutline color="gold.500" />
+            <Text
+              textStyle="text-sm-mono-medium"
+              color="gold.500"
+            >
+              Favorites
```
Comment (herbig) [inline src/components/menus/FavoritesMenu/index.tsx:20]: should use `titleFavorites` here

**Review by herbig — CHANGES_REQUESTED** [review]

**Comment by Da-Colon** [comment]
> @Da-Colon is there a way I can set a DAO as a favorite in this branch?

Not yet. I believe this will be added on the Dashboard when thats complete

File: src/components/menus/FavoritesMenu/index.tsx (hunk 2 above)
Comment (Da-Colon) [reply src/components/menus/FavoritesMenu/index.tsx]: done.

**Pull request merged**